### ISSUE TITLE
Include git best practices?

### ISSUE THREAD

**Issue by MicahGale** [cnerg/py-template#10]
Things like:

1. "update" is not a commit message
2. 1,000 line commits are not proper
3. You're on the wrong side of the internet if they are telling you to do `git reset -f`

Not sure if this the proper place for it though.

**Comment by gonuke** [comment]
I do think we want to capture some of these things, but not sure where.  This effort started with [this](https://docs.google.com/document/d/1ReIEbjjCTk6wjsSJtuFq8nGUctrPdchYbJDbWmFXSbM/edit#heading=h.7kz70yolvbja), but then I decided to put some of it in a repo.  Not sure where the balance will reside at the end.  Could also be added to CNERG website.

**Comment by MicahGale** [comment]
Yeah... I'm not sure about where the best place for this is. There is a lot of advice online, but not a lot of it seems to cover just: what is a good commit. There is this: https://sethrobertson.github.io/GitBestPractices/, but it's very verbose.

**Comment by MicahGale** [comment]
I'm thinking of just adding a few quick bullet points on git resources, and if needed making a wiki page in this repo?

**Comment by MicahGale** [comment]
@lewisgross1296, didn't we cover these guidelines in your guide on Google Drive?

**Comment by lewisgross1296** [comment]
A good amount of it probably but not all. I'll leave it up to @MicahGale to decide if I'm missing anything you intended to be in there
